bin/cml-send-comment.js

```diff
@@ -11,7 +11,7 @@ const run = async (opts) => {
   const path = opts._[0];
   const report = await fs.readFile(path, 'utf-8');
   const cml = new CML(opts);
-  await cml.commentCreate({ ...opts, report });
+  console.log(await cml.commentCreate({ ...opts, report }));
```

Comment:
Now the `cml-send-comment` command will return a link to the comment for consistency with tools like `cml-pr` or `gh` that already do that. It seems like a good addition to me, but is a bit out of scope for this pull request.

Reply:
Its printing in stderror. It should have 
const print = console.log